pkg/helm/cluster/kube_version.go

```diff
@@ -0,0 +1,30 @@
+package cluster
+
+import (
+	"errors"
+	"helm.sh/helm/v3/pkg/chartutil"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+)
+
+func GetKubeVersion(config *rest.Config) (*chartutil.KubeVersion, error) {
```

Comment:
should not we remove this function here, since you have introduced one in `actions` package?

Reply:
done